fast_api_als/routers/lead_conversion.py

```diff
@@ -65,7 +65,11 @@ async def submit(file: Request, token: str = Depends(get_token)):
     if is_updated:
         logging.info("Lead conversion updated")
```

Comment:
not the way to log across modules. Use a custom logger (specifying module name) to get logs that'd be useful later on.